public/data/tibebeSol.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Tibebe Solomon",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!, worked on C, Java and Python",
+  "avatar": "https://github.com/sapencio.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/sapencio",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/tibebesolo",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Freelancing",
+      "date": "Septembe 2022",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started freelancing",
+      "url": "https://linkfree.eddiehub.io/sapencio"
```

Comment:
I think this would be `tibebeSol` due to your filename.
```suggestion
      "url": "https://linkfree.eddiehub.io/tibebeSol"
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable